target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -103,6 +103,11 @@ netgear,r7450)
 	ucidef_set_led_netdev "lan3" "LAN3" "white:lan3" "lan3"
 	ucidef_set_led_netdev "lan4" "LAN4" "white:lan4" "lan4"
 	;;
+netgear,wax202)
+	ucidef_set_led_netdev "internet" "Internet" "green:net" "wan"
+	# NOTE: hardware already blinks green LEDs for lan1-lan3,
```

Comment:
Does this mean there are switch controlled LEDs?

Reply:
I believe so, at least the LEDs blink without any configuration

Reply:
That means you can add configuration if you want.

Reply:
Do you mean configuring the LEDs to be gpio instead of switch-controlled and then configuring them in this file? Would that be preferred?

Reply:
Sure. I personally turn off all LEDs. Doing that would allow doing so.

Reply:
@looi news with this?

Reply:
Done